README.md

```diff
@@ -896,6 +896,16 @@ Note that the Circus Train configuration only supports one set of AWS credential
           table-name: aws-a_table
           table-location: s3://<aws-a-data-bucket>/<pata-to-aws-a_db>/aws-a_table
 
+## Circus Train extensions
+
+### Circus Train DataSqueeze Copier
+This extension implements new Copiers for Circus Train which integrate DataSqueeze.
```

Comment:
Could you reword this as "The DataSqueeze extension provides new Copiers for Circus Train which compact the data being replicated in various ways" and then make the word "DataSqueeze" a hyperlink to the URL instead of listing it in plain text.

Reply:
Done